src/common/tusb_mcu.h

```diff
@@ -298,6 +298,13 @@
   #define TUP_USBIP_FSDEV_STM32
   #define TUP_DCD_ENDPOINT_MAX    8
 
+#elif TU_CHECK_MCU(OPT_MCU_STM32H7RS)
+  #define TUP_USBIP_DWC2
+  #define TUP_USBIP_DWC2_STM32
+
+  // FS has 6, HS has 9
```

Comment:
stm32h7rs has built-in HS PHY, we should add 
```
#define TUP_RHPORT_HIGHSPEED    1
```

as well, since people will also prefer to use that port.

Reply:
Done